tests/setuponlytests/spigot_world_for_spigot_server/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+  mc:
+    restart: "no"
+    image: ${IMAGE_TO_TEST:-itzg/minecraft-server}
+    environment:
+      EULA: "TRUE"
+      SETUP_ONLY: "TRUE"
+      VERSION: ${MINECRAFT_VERSION:-LATEST}
+      TYPE: "PAPER"
```

Comment:
I need to know more of why we are using paper instead of spigot when that is declared in the folder name.

Reply:
Spigot is effectively a dead release stream; however, it lives on as a "family" designation where Paper is a member of that family.